client/redis/redis_test.go

```diff
@@ -2,12 +2,12 @@ package redis
 
 import (
 	"context"
-	"testing"
-
+	"github.com/alicebob/miniredis/v2"
 	"github.com/beatlabs/patron/trace"
 	"github.com/opentracing/opentracing-go"
 	"github.com/opentracing/opentracing-go/mocktracer"
 	"github.com/stretchr/testify/assert"
+	"testing"
```

Comment:
import ordering seems off. you should use gofumpt or goimport for this. you can even automate this but this depends on your ide.

Reply:
Yes, Thanks for pointing out. I enabled run goimport on save now. Fixed the ordering